pkg/console/subresource/configmap/configmap.go

```diff
@@ -18,23 +19,57 @@ const (
 	consoleConfigYamlFile   = "console-config.yaml"
 	clientSecretFilePath    = "/var/oauth-config/clientSecret"
 	oauthEndpointCAFilePath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-	// TODO: should this be configurable?  likely so.
-	documentationBaseURL = "https://docs.okd.io/4.0/"
-	brandingDefault      = "okd"
 	// serving info
 	certFilePath = "/var/serving-cert/tls.crt"
 	keyFilePath  = "/var/serving-cert/tls.key"
 )
 
-func DefaultConfigMap(cr *operatorv1.Console, rt *routev1.Route) *corev1.ConfigMap {
+// overridden by console config
+const (
+	defaultLogoutURL = ""
+)
+
+// overridden by operator config
+const (
+	defaultDocumentationBaseURL = "https://docs.okd.io/4.0/"
+	defaultBranding             = "okd"
+)
+
+//consoleConfig.Spec.Authentication.LogoutRedirect
+func getLogoutRedirect(consoleConfig *configv1.Console) string {
+	// TODO: handle if not provided
+	// return consoleConfig.Spec.Authentication.LogoutRedirect || defaultLogoutURL
+	return consoleConfig.Spec.Authentication.LogoutRedirect
+}
+
+//operatorConfig.Spec.Customization.Brand
+func getBrand(operatorConfig *operatorv1.Console) operatorv1.Brand {
+	// TODO: handle if not provided
```

Comment:
@zherman0 this is related to your brand work PR.

Reply:
#126 
Just want to ensure they function together.

Reply:
So, if brand is not EXPLICITLY set, it should just fallback to the default which is provided in #126.